Dangerfile.swift

```diff
@@ -3,17 +3,17 @@ import Danger
 
 let danger = Danger()
 
-// if danger.github.pullRequest.branch_for_base != "develop" {
+// if danger.git.branch_for_base != "develop" {
 //     fail("Please re-submit this PR to develop, we may have already fixed your issue.")
 // }
 
 // Pull request size
-let bigPRThreshold = 200
+let bigPRThreshold = 600
 let additions = danger.github.pullRequest.additions!
 let deletions = danger.github.pullRequest.deletions!
 let changedFiles = danger.github.pullRequest.changedFiles!
 if (additions + deletions > bigPRThreshold) {
-    warn("PR size seems relatively large. ✂️ If this PR contains multiple changes, please split each into separate PR will helps faster, easier review.")
+    fail("PR size seems relatively large. ✂️ If this PR contains multiple changes, please split each into separate PR will helps faster, easier review.")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/dbgarcia/danger-bot/blob/4ad3f65193926f1211a9cfb888b46cbbba684b48/Dangerfile.swift#L16"></span>Line should be 130 characters or less: currently 152 characters<br />
<code>line_length</code> <code>Dangerfile.swift:16</code></del></td>
    </tr>
  </tbody>
</table>